### PR TITLE
Updated Header and associated styles

### PR DESCRIPTION
**Pull request by wilwade**
Problem
=======

- Updating to the latest header design
- Small tweaks to other colors and the footer
- Fixed header margins and other tailwind collisions

Screenshots (optional):
-----------------------
image

image

image

### CODE REVIEW

File: css/header.css [hunk 1]
```diff
@@ -1,138 +1,3 @@
...
-  font-family: "Newake", "Arial Black", Arial, sans-serif;
-  text-transform: uppercase;
-  color: var(--header-color-primary);
-  display: flex;
-  justify-content: space-between;
-  align-items: center;
-  box-sizing: border-box;
-
-  background-color: var(--header-background);
+.header-shadow {
```
Comment (wilwade) [inline css/header.css:1]: The one we still need. The rest are tailwind

File: theme/css/chrome.css [hunk 2]
```diff
@@ -191,23 +191,23 @@ a > .hljs {
   margin-block-start: 50px;
 }
 
-.mobile-nav-chapters {
+.content .mobile-nav-chapters {
```
Comment (wilwade) [inline theme/css/chrome.css:194]: CSS specificity is fighting a bit with the tailwind styles

File: theme/css/chrome.css [hunk 3]
```diff
@@ -191,23 +191,23 @@ a > .hljs {
...
   display: block;
   font-size: 1em;
   text-align: center;
   text-decoration: none;
   padding: 10px 20px;
   border-radius: 5px;
-  background-color: var(--page-nav-bg);
-  color: var(--links);
+  background-color: var(--button-nav-bg);
+  color: var(--button-nav-fg);
```
Comment (wilwade) [inline theme/css/chrome.css:202]: Fixing button colors to not hide text

File: theme/css/chrome.css [hunk 4]
```diff
@@ -471,6 +471,7 @@ ul#searchresults span.teaser em {
 .sidebar .sidebar-scrollbox {
   position: sticky;
   top: 80px;
+  padding: 30px 0;
```
Comment (wilwade) [inline theme/css/chrome.css:474]: Adding some padding to the top and bottom of the sidebar

File: theme/css/general.css [hunk 5]
```diff
@@ -63,7 +63,7 @@ h6 code {
   display: none;
 }
 .hidden {
-  display: none !important;
+  display: none;
```
Comment (wilwade) [inline theme/css/general.css:66]: Cannot be important due to tailwind collision

File: theme/css/general.css [hunk 6]
```diff
@@ -155,8 +155,9 @@ h6:target::before {
 
 .content {
   overflow-y: auto;
-  padding: 0 15px;
+  padding: 30px 15px;
   padding-bottom: 50px;
+  min-height: 75vh;
```
Comment (wilwade) [inline theme/css/general.css:160]: This makes sure that the content size is always at least most of the way

File: theme/css/general.css [hunk 7]
```diff
@@ -180,7 +181,7 @@ h6:target::before {
   line-height: 1.45em;
 }
 .content a {
-  text-decoration: none;
+  text-decoration: underline;
```
Comment (wilwade) [inline theme/css/general.css:184]: Links were getting lost with the low contrast highlight

File: theme/css/variables.css [hunk 8]
```diff
@@ -6,21 +6,21 @@
   --sidebar-resize-indicator-width: 4px;
   --sidebar-resize-indicator-space: 2px;
   --page-padding: 15px;
-  --content-max-width: 950px;
+  --content-max-width: 1080px;
```
Comment (wilwade) [inline theme/css/variables.css:9]: More space for the various padding that was added

File: theme/css/variables.css [hunk 9]
```diff
@@ -6,21 +6,21 @@
...
   --mono-font: "Source Code Pro", Consolas, "Ubuntu Mono", Menlo, "DejaVu Sans Mono", monospace, monospace;
   --code-font-size: 0.875em /* please adjust the ace font size accordingly in editor.js */;
-  --header-height: 80px;
+  --header-height: 85px;
 }
 
 /* Themes */
 
 .coal {
-  --bg: #0b1e28;
```
Comment (wilwade) [inline theme/css/variables.css:19]: Fixed lots of colors with the new dark blue

File: theme/header.hbs [hunk 10]
```diff
@@ -1,68 +1,95 @@
...
-          fill="currentColor"
-        />
-      </svg>
-    </a>
+<header id="header" class="bg-white header-shadow flex h-[85px] font-title sticky left-0 top-0 w-full z-10">
```
Comment (wilwade) [inline theme/header.hbs:1]: Mostly copy paste from the header in storybook

File: theme/css/chrome.css (hunk 2 above)
Comment (shannonwells) [reply theme/css/chrome.css:194]: as per usual

**Review by shannonwells — APPROVED** [review]
LGTM

**Pull request merged**